pace-util/pace/util/null_comm.py

```diff
@@ -86,3 +86,6 @@ def Split(self, color, key):
             comm.total_ranks = total_ranks
         self._split_comms[color].append(new_comm)
         return new_comm
+
+    def allreduce(self, sendobj, op=None) -> Any:
+        return self._fill_value
```

Comment:
```suggestion
    def allreduce(self, sendobj, op=None) -> Any:
        return sendobj
```
nit: data doesn't matter for null comms, but this would be less likely to cause an exception.